EthDefaults.cmake

```diff
@@ -0,0 +1,21 @@
+# it must be a macro, cause 'include_directories' 
+# are not propagated to parent scope
+macro(eth_defaults)
+	set(ETH_DIR             "${CMAKE_CURRENT_LIST_DIR}/../cpp-ethereum"         CACHE PATH "The path to the cpp-ethereum directory")
+	set(ETH_BUILD_DIR_NAME  "build"                                             CACHE STRING "The name of the build directory in cpp-ethereum")
+	set(ETH_BUILD_DIR       "${ETH_DIR}/${ETH_BUILD_DIR_NAME}")
+	set(SOL_DIR             "${CMAKE_CURRENT_LIST_DIR}/../solidity"             CACHE PATH "The path to the solidity directory")
```

Comment:
This is also set in UseSolidity.cmake. I'd suggest leaving it there since it is not required anywhere else